kolibri/core/assets/src/views/KTooltip.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+
+  <popper
+    :disabled="disabled"
+    :visibleArrow="false"
+    :options="options"
+    trigger="hover"
+    class="container d-ib"
+  >
+    <div
+      slot="reference"
+      class="d-ib"
```

Comment:
Curious what the purpose of this is?

It seems potentially brittle for us to be overriding the display style of reference elements.